documentation/jetty/modules/operations-guide/pages/deploy/index.adoc

```diff
@@ -133,24 +142,31 @@ core-app
 │   └── com
 │       └── acme
 │           └── AcmeHandler.class
-└── lib            # Where web application Java libraries reside.
-    └── acme-util.jar
+├── lib            # Where web application Java libraries reside.
+│   └── acme-util.jar
+└── jetty-web.xml  # Web application specific configuration.
```

Comment:
This file is now badly named. It is nothing to do with `web.xml` but rather another Jetty XML context XML file that is processed after `web.xml`.

Reply:
Also, how are you arranging for this to be processed for a core application, where there is no concept of `Configuration`s  that would read such a file?